tracer/src/Datadog.Trace/SpanContext.cs

```diff
@@ -44,6 +45,14 @@ public SpanContext(ulong? traceId, ulong spanId, SamplingPriority? samplingPrior
         {
             SpanId = spanId;
             SamplingPriority = (int?)samplingPriority;
+
```

Comment:
Should this be moved to the private .ctor to avoid adding a new one and forgetting this ? I assume there are no cases where `CIVisibility.IsRunning` is true and we don't want to set that origin.

Reply:
done in 3324c8d